stetho/src/main/java/com/facebook/stetho/inspector/network/ResponseBodyFileManager.java

```diff
@@ -56,19 +73,44 @@ public ResponseBodyData readFile(String requestId) throws IOException {
       }
       ResponseBodyData bodyData = new ResponseBodyData();
       bodyData.base64Encoded = firstByte != 0;
-      bodyData.data = readContentsAsUTF8(in);
+
+      if (mRequestIdMap.containsKey(requestId)) {
```

Comment:
This is also invalid synchronization.  `get` is all you need to call.  It is faster, simpler, and is thread safe.